source/slang/slang-emit-c-like.cpp

```diff
@@ -1849,6 +1882,27 @@ void CLikeSourceEmitter::emitIntrinsicCallExprImpl(
     }
 }
 
+void CLikeSourceEmitter::_emitCallArgList(IRCall* inst)
+{
+    m_writer->emit("(");
+    UInt argCount = inst->getOperandCount();
+    for (UInt aa = 1; aa < argCount; ++aa)
+    {
+        auto operand = inst->getOperand(aa);
+        if (as<IRVoidType>(operand->getDataType()))
+            continue;
+
+        // TODO: [generate dynamic dispatch code for generics]
+        // Pass RTTI object here. Ignore type argument for now.
+        if (as<IRType>(operand))
+            continue;
+
+        if (aa != 1) m_writer->emit(", ");
```

Comment:
How do these `continue` cases work together with the `a != 1` logic for emitting `,`? It seems like if the first argument is `void` or a type then we won't emit anything, and then the next argument (which is the first *actual* argument) will start out by emitting a `,`.